src/App.css

```diff
@@ -1,31 +1,93 @@
-header,
-ul,
-li {
+* {
+  font-family: 'Lato', sans-serif;
+}
+
+header {
+  border: 3px solid #e8e8e8;
   display: flex;
-  margin-right: 20px;
-  justify-content: end;
-  align-items: center;
+  flex-direction: row;
+  height: 10vh;
+}
+
```

Comment:
- Please stick to the design of the Bookstore

Your design:
![Screen Shot 2022-01-29 at 16 33 10](https://user-images.githubusercontent.com/67211919/151679958-1367b0d1-86e6-4147-a06f-fb34591895f8.png)

The design we want to achieve:
![Screen Shot 2022-01-29 at 16 33 41](https://user-images.githubusercontent.com/67211919/151679960-468ecfba-30e9-4422-9257-c749c4301e1f.png)

 * Stick to the design as much as possible